libs/coin-modules/coin-tezos/src/logic/listOperations.ts

```diff
@@ -47,7 +47,19 @@ export async function listOperations(
   const nextToken = lastOperation ? JSON.stringify(lastOperation?.id) : "";
   const filteredOperations = limitedOperations
     .filter(op => isAPITransactionType(op) || isAPIDelegationType(op) || isAPIRevealType(op))
-    .reduce((acc, op) => acc.concat(convertOperation(address, op)), [] as Operation[]);
+    .filter(op => {
+      // Filter out failed incoming transactions as they don't affect the account balance
+      const hasFailed = op.status && op.status !== "applied";
```

Comment:
```suggestion
      const hasFailed = op.status !== "applied";
```
`undefined !== "applied"` is `true`